### DISCUSSION TITLE
What is  composer.json in wave folder?

### DISCUSSION THREAD

**Discussion by dyaskur** [thedevdojo/wave#160] [Q&A]
What if I don't install composer on that path?

**Comment by bobbyiliev** [comment]
Hey! 

Yep, it is not needed to run composer install in that path! You can ignore that compose json file.

**Reply by dyaskur** [reply]
I noticed that there are some code/files that don’t seem to be in use. Would it make sense to clean them up or remove them if they’re no longer needed? Because it might be confusing to developers.

**Reply by bobbyiliev** [reply]
Yes, good point! Probably some files might have been left from Wave v2. I will do a scan and try to remove any that are not needed in v3!
